### DISCUSSION TITLE
katex.min.js无法在国内请求到

### DISCUSSION THREAD

**Discussion by Kuangdd01** [ppoffice/hexo-theme-icarus#1082] [General]
katex.min.js无法在国内请求到，应该要在哪个文件内修改src引用呢？

**Comment by ppoffice** [comment]
https://ppoffice.github.io/hexo-theme-icarus/Configuration/icarus%E7%94%A8%E6%88%B7%E6%8C%87%E5%8D%97-cdn%E6%8F%90%E4%BE%9B%E5%95%86/

**Reply by Kuangdd01** [reply]
thx！！
